app/controllers/saml_idp_controller.rb

```diff
@@ -38,6 +36,22 @@ def logout
 
   private
 
+  def process_fully_authenticated_user
+    link_identity_from_session_data
+
+    needs_idv = identity_needs_verification?
+    analytics.track_event(Analytics::SAML_AUTH, @result.to_h.merge(idv: needs_idv))
+
+    yield true if needs_idv
+
+    yield false
```

Comment:
That was not intentional, but it doesn't seem to cause a problem. However, I'll fix it so that it reads `yield needs_idv`. That's what I get for coding late at night.